_cxx/envoy.mk

```diff
@@ -206,7 +208,7 @@ $(OSS_HOME)/docker/base-envoy/envoy-static: $(ENVOY_BASH.deps) FORCE
 	            exit 1; \
 	        fi; \
 	        $(call ENVOY_BASH.cmd, \
-	            $(ENVOY_DOCKER_EXEC) bazel build --verbose_failures -c $(ENVOY_COMPILATION_MODE) --config=clang //source/exe:envoy-static; \
+	            $(ENVOY_DOCKER_EXEC) bazel build $$(if [ -n '$(FIPS_MODE)' ]; then echo "--define boringssl=fips"; fi) --verbose_failures -c $(ENVOY_COMPILATION_MODE) --config=clang //source/exe:envoy-static; \
```

Comment:
This can be better written with Make macros: `$(if $(FIPS_MODE), --define boringssl=fips )`.

Reply:
Thanks for the feedback, not super familiar with using make macros, but tested with the change and it did infact set it correctly!